torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -847,5 +853,16 @@ std::vector<std::string> export_opnames(const script::Module& m) {
   return std::vector<std::string>(names.begin(), names.end());
 }
 
+// Thread local floag (only happens in export, i.e. on server side)
+// to control if
```

Comment:
to control if ...?

maybe add more detail what it controls.

Reply:
Will update it. Thanks!